internal/vds/vds_test.go

```diff
@@ -315,3 +316,98 @@ func TestSliceMetadataAxisOrdering(t *testing.T) {
 		}
 	}
 }
+
+func TestFence(t *testing.T) {
+	expected := []float32{
+		108, 109, 110, 111, // il: 3, xl: 10, samples: all
+		112, 113, 114, 115, // il: 3, xl: 11, samples: all
+		100, 101, 102, 103, // il: 1, xl: 10, samples: all
+        108, 109, 110, 111, // il: 3, xl: 10, samples: all
+        116, 117, 118, 119, // il: 5, xl: 10, samples: all
+	}
+
+	testcases := []struct{
+		coordinate_system string
+		coordinates       [][]float32
+	} {
+		{
+			coordinate_system: "ij",
+			coordinates:
+				[][]float32{{1, 0}, {1, 1}, {0, 0}, {1, 0}, {2, 0}},
+		},
+		{
+			coordinate_system: "ilxl",
+			coordinates:
+				[][]float32{{3, 10}, {3, 11}, {1, 10}, {3, 10}, {5, 10}},
```

Comment:
Since we are just returning whatever upenvds gives us also in this case I say we leave it as unspecified behaviour for the time being. We might want to do something different than openvds later on.

Reply:
This is still why I ask for tests in all these cases :) 
Write a placeholder for them, mark them as failing for "unspecified" behavior. Literally just test name and failure reason, to mark that yes, this is a valid problem.
If you don't want to do so, then create issues for boundaries and for interpolation.
But lets not put our heads in the sand and pretend we never saw it.

(Though just note that slice already checks for boundaries.)

Reply:
Made issues for this.

Reply:
Go rebase then and I'll do a final look.